docs/section5/Rule5-48.md

```diff
@@ -2,8 +2,8 @@
 # Envelope - Rule 5-48  
 
 **Rule ID:** 5-48  
-**Rule Description:** The air leakage rate, schedule and method in unconditioned and unenclosed spaces must be the same in the baseline and proposed design.  
-**Rule Assertion:** B-RMR infiltration:infiltration.air_leakage_rate, modeling_method, modeling_method_name, multiplier_Schedule= P-RMR infiltration.air_leakage_rate, modeling_method, modeling_method_name, multiplier_Schedule  
+**Rule Description:** The air leakage rate must be the same in the baseline and proposed design.  
```

Comment:
The rule description should say "The air leakage rate in spaces that are neither conditioned nor semiheated must be the same in the baseline and proposed design."

Reply:
Updated to unconditioned or unenclosed as per the previous TCD description.